### DISCUSSION TITLE
Make ubuntu-latest point to ubuntu-22.04

### DISCUSSION THREAD

**Discussion by famastefano** [actions/virtual-environments#6257] [Ideas]
Took me a little bit to understand why my GitHub Actions couldn't find clang-14, and then I saw that I was running on Ubuntu 20.04, while 22.04 is available.
Shouldn't the *-latest runner point to the latest available image? Otherwise what's the point of calling it like that?
Or is it for backward compatibility?
If not, I suggest to update it to the latest runner, like the name implies.
If someone cares about the exact OS version, they should explicitly state that by using a versioned runner name.

**Comment by panekj** [comment]
https://github.com/actions/runner-images/issues/5998#issuecomment-1231939134
